resources/assets/templates/vote/voteCreateLayout.hbs

```diff
@@ -74,11 +32,13 @@
     <div class="vote-new-access">
         <div class="form-group well">
             <label for="usr">Accessed:</label>
+            <button type="button" class="btn btn-default btn-info js-to-accessed" style="float: right;">Move all users to accessed</button>
```

Comment:
Please don't use styles in tags. You can use bootstrap style class 'pull-right' instead